gstreamer/src/query.rs

```diff
@@ -15,188 +15,188 @@ use std::ptr;
 use std::mem;
 use std::fmt;
 use std::ffi::CStr;
-use std::ops::Deref;
+use std::ops::{Deref, DerefMut};
 
 use glib;
 use glib::translate::{from_glib, from_glib_full, ToGlib, ToGlibPtr};
 
 #[repr(C)]
-pub struct QueryRef(ffi::GstQuery);
+pub struct Query(ffi::GstQuery);
```

Comment:
Also generally these things are currently called `XXXRef` and the owned thing is called `XXX`. See `Structure`, `VideoFrame`. Idea here being similar to having `Path` and `PathBuf`, consistently named.

Reply:
generally, I'm not too happy with the naming but we can fix that later :)

Reply:
Ok, I'll revert this.